### PR TITLE
Fix example for mc admin bucket quota

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: cmd/admin-bucket-quota.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ EXAMPLES:
 	  {{.Prompt}} {{.HelpName}} myminio/mybucket --fifo 64kB
```
Comment (kannappanr) [inline cmd/admin-bucket-quota.go:103]:
```suggestion
	  {{.Prompt}} {{.HelpName}} myminio/mybucket --fifo 64KB
```

Comment (harshavardhana) [inline cmd/admin-bucket-quota.go:103]: isn't this also inconsistent?

Comment (poornas) [reply cmd/admin-bucket-quota.go:103]: this is the convention to write kilo bytes afaics.

Comment (harshavardhana) [reply cmd/admin-bucket-quota.go:103]: We should support both styles @poornas also do we really care about kilobytes fifo? Are we trying to show that you can only store 1 file? 

Examples can be a bit more geared towards practical usecases.

**Pull request merged**